pkgs/top-level/all-packages.nix

```diff
@@ -240,6 +240,8 @@ with pkgs;
 
   alire = callPackage ../development/tools/build-managers/alire { };
 
+  alp = callPackage ../by-name/al/alp/package.nix { };
+
```

Comment:
```suggestion
```

The by-name hierarchy uses an automatic registration.

Reply:
ah cool, good to know!